src/website/interface.ts

```diff
@@ -221,6 +238,21 @@ export class WebsiteInterface {
 			// we've set discord IDs to this
 			await this.setSeed(tournamentId, byePlayers[i], oppSeed);
 		}
+
+		// detailed logging
+		// update array after challonge changes. REMOVE AFTER LOGS NOT NEEDED
+		const newPlayers = await this.api.getPlayers(tournamentId);
```

Comment:
I was going to say that we could save this for later but we got rid of automatic round 1 pairings.